doc/RELEASE_WALKTHROUGH.rst

```diff
@@ -79,7 +79,7 @@ Generate the changelog
 
 The changelog is generated using the changelog tool::
 
-    $ python tools/changelog.py $GITHUB v1.20.0..maintenance/1.21.x > doc/changelog/1.21.0-changelog.rst
+    $ spin changelog -t $GITHUB --revision-range v1.20.0..maintenance/1.21.x > doc/changelog/1.21.0-changelog.rst
```

Comment:
I haven't tried this yet, but are `-t` and `--revision-range` optional of required?

Reply:
It's required. The script it calls downstream need the args: https://github.com/numpy/numpy/blob/44efd35ab74b8aac6b580a927b0d540344551610/tools/changelog.py#L109

We can make the token a default value env variable, but the revision must be required. We can infer it however to default to n-1 and nth release, where n is latest.

Reply:
The script specifies them as positional arguments. Do the spin commands allow that?

Reply:
Yep, indeed. We need to change `option` to `argument`. I have made the changes.